user/id_test.go

```diff
@@ -58,6 +59,31 @@ func TestID_ReadFromV2(t *testing.T) {
 	})
 }
 
+func TestID_ScriptHash(t *testing.T) {
```

Comment:
not needed, `TestNewFromScriptHash` already exists

Reply:
Removed